postman/vets-api.pm-collection.json

```diff
@@ -756,118 +756,6 @@
 					},
 					"response": []
 				},
-				{
```

Comment:
This looked like a recent file. I imagine that it is platform-owned, but I tried to help out by removing the two VA Forms API endpoints that no longer live in `vets-api` due to this PR removing them.